src/pages/settings/PasswordPage.js

```diff
@@ -139,15 +195,25 @@ class PasswordPage extends Component {
                                 autoCompleteType="password"
                                 textContentType="password"
                                 value={this.state.newPassword}
-                                onChangeText={newPassword => this.setState({newPassword})}
-                                onFocus={() => this.setState({isPasswordRequirementsVisible: true})}
-                                onBlur={() => this.onBlurNewPassword()}
+                                hasError={this.state.errors.newPassword || this.state.errors.newPasswordSameAsOld}
+                                errorText={!this.state.errors.newPassword && this.state.errors.newPasswordSameAsOld
```

Comment:
Do we need to check both of these conditions to determine which errorText to show? If there's no error it will not return anything?